desktop/libs/azure/src/azure/abfs/upload.py

```diff
@@ -53,17 +55,17 @@ def __init__(self, request, *args, **kwargs):
     self.totalfilesize = kwargs.get('qqtotalfilesize')
     self.file_name = kwargs.get('qqfilename')
     if self.file_name:
-      self.file_name = unicodedata.normalize('NFC', self.file_name) # Normalize unicode
+      self.file_name = unicodedata.normalize('NFC', self.file_name)  # Normalize unicode
     self.destination = kwargs.get('dest', None)  # GET param avoids infinite looping
     self.target_path = None
 
-    if kwargs.get('chunk_size', None) != None:
+    if kwargs.get('chunk_size', None) is not None:
```

Comment:
Can simplify this like for some other review comment above. Something like `if kwargs.get('chunk_size')`?

Reply:
Yes, removed this check